PluginsAndFeatures/azure-toolkit-for-intellij/azure-intellij-plugin-azure-sdk/src/main/java/com/microsoft/azure/toolkit/intellij/azure/sdk/buildtool/StorageUploadWithoutLengthCheck.java

```diff
@@ -0,0 +1,108 @@
+package com.microsoft.azure.toolkit.intellij.azure.sdk.buildtool;
```

Comment:
Where are the tests? There's a bit of logic here we should make sure works.

Reply:
We're manual testing for now

Reply:
This is what I meant: https://gist.github.com/conniey/28b8a35b1ac7a1e4930d47e086a73f34